receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_constants.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpuscraper
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/consumer/pdata"
+)
+
+// cpu metric constants
+
+var (
+	StateLabel = "state"
+	CPULabel   = "cpu"
+)
+
+var (
+	UserStateLabelValue      = "user"
+	SystemStateLabelValue    = "system"
+	IdleStateLabelValue      = "idle"
+	InterruptStateLabelValue = "interrupt"
+	IowaitStateLabelValue    = "iowait"
+)
```

Comment:
Do we have somewhere documented why do we go with labels for state vs a different metric per state?

Reply:
The reason for that choice isn't explicitly documented, but we're working off requirements in this doc: [Host Metrics](https://docs.google.com/document/d/1i-QG-TxkIWVbdx86UM2GXcFvpeCV7zSBsPmMyMkOEBk/edit) - feel free to add comments.

I figure these are essentially different categories of cpu time, so make sense to be labels.